tests/functional/README.md

```diff
@@ -0,0 +1,49 @@
+# Casper JS tests
+
+## Initial setup (one time)
+
+### Mac OS
+
+    cd test/casper
+
+Install [Docker Toolbox](https://www.docker.com/products/docker-toolbox)
```

Comment:
These instructions for installing and starting docker are out of date, but I don't think they're necessary here anymore. This section (Initial Setup) and th instructions in Running the tests can go.